src/supervisor.sbvr

```diff
@@ -0,0 +1,461 @@
+
+	Fact type: application has uuid
+	-- Optional: wait for v3 target state?
+		Necessity: each application has exactly one uuid.
+		Necessity: each uuid is of exactly one application.
+		Necessity: each application has a uuid that has a Length (Type) that is equal to 32.
+
+	Fact type: application is host
+
+	Fact type: application is healthy
+	-- GET /v1/healthy
+		Necessity: each application that has a slug that is equal to "balena-os/balena-supervisor" is healthy or not healthy.
+		-- Not sure about syntax / validity for "healthy" boolean
+	
+	Fact type: application has download progress
+	-- Maybe should be computed term of all image download progresses?
+		Necessity: each application has at most one download progress.
+
+	Fact type: application should be running release
+		Synonymous Form: release should be running on application
+		Necessity: each application should be running at most one release.
+
+	Fact type: application has version
+		Necessity: each application that has a slug that is equal to "balena-os/balena-supervisor" has exactly one version.
+
+	Fact type: application has api secret
+		Necessity: each application that has a slug that is equal to "balena-os/balena-supervisor" has at least one api secret.
+
+	Fact type: device belongs to application
+		Synonymous Form: application owns device
+		Necessity: each device belongs to at least one application.
+		-- Changed from "at most one application" to "at least" for multi-app -- valid?
+
+	Fact type: application has commit
+		Necessity: each application has exactly one commit.
+		Necessity: each commit belongs to exactly one application.
+		Necessity: each commit has a uuid that has a Length (Type) that is equal to 32.
+
+
+
+-- CONFIG / ENV VARS
+Term: value
+	Concept Type: Text (Type)
+
+Term: config var
+	Concept Type: Short Text (Type)
+
+	Fact type: config var has value
+		Necessity: each config var has exactly one value.
+
+	Fact type: application owns config var
+		Synonymous Form: config var belongs to application
+		Necessity: each application that has a config var must have a slug equal to "balena-os/balena-supervisor".
+		Necessity: each config var that belongs to an application must begin with "BALENA_" or "RESIN_".
+
+Term: env var
+	Concept Type: Short Text (Type)
+
+	Fact type: env var has value
+		Necessity: each env var has exactly one value.
+
+	Fact type: service owns env var
+		Synonymous Form: env var belongs to service
+
+
+
+-- UPDATE LOCK
+
+Term: update lock
+	Concept Type: Short Text (Type)
+	-- Path of update lockfile (Short Text (Type))?
+	-- Or integer written into lockfile (Integer (Type))?
+
+	Fact type: application has update lock
+		Necessity: each application has at most one update lock.
+
+	Fact type: service has update lock
+		Necessity: each service has at most one update lock.
+
+
+
+-- ACTION: TODO
+-- Since this is theoretical, this probably shouldn't initially be included in the model.	
```

Comment:
Yes, I think we should take it out, let's see how far we can get with data first

Reply:
Done